src/cmd/initialize.go

```diff
@@ -58,16 +59,23 @@ var initCmd = &cobra.Command{
 					message.Fatalf(nil, "This command requires a zarf-init package, but one was not found on the local system.")
 				}
 
-				// If no CLI version exists (should only occur in dev or CI), try to get the latest release tag from Githhub
-				if _, err := semver.StrictNewVersion(config.CLIVersion); err != nil {
-					config.CLIVersion, err = utils.GetLatestReleaseTag(config.GithubProject)
+				// Parse the CLI version and extract its parts
+				initPackageVersion := strings.TrimLeft(config.CLIVersion, "v")
```

Comment:
Perhaps we should just do `strings.Trim()` for consistency instead here since the source left whitespace issue is resolved?

Reply:
Here we are trimming the `v` out and only expect it to be at the front of the string.  We could do a more precise version of this with something like:

```
if len(config.CLIVersion > 0) {
  initPackageVersion := config.CLIVersion[1 : len(config.CLIVersion)-1]
}
```

It felt simpler to use `strings.TrimLeft` even though that allows `vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv0.10.0`

Reply:
fair enough.  Since we don't have a test for this unique usecase (and it would be hard to test I think), can you drop a screenshot of it working as expected now?

Reply:
![image](https://user-images.githubusercontent.com/3977569/183523338-d2812d90-ca81-487d-8e0f-e60c5180ca96.png)

Here are most cases.  Top to bottom:

That branch as it is
A valid main release
An invalid semantic version
A valid older git commit